### DISCUSSION TITLE
How can render custom html in this plugin AKA pass the html wrap in sed link and display it in the lightbox.

### DISCUSSION THREAD

**Discussion by BAProductions** [archetyped/simple-lightbox#1080] [Q&A] [answered]
How can render custom html in this plugin AKA pass the html wrap in sed link and display it in the lightbox.

**Comment by archetyped** [comment, marked as answer]
There is an [extension proposal for HTML support](https://github.com/archetyped/simple-lightbox-extensions/issues/122).  Adding your vote to that ticket is recommended if you would like an extension to add this functionality.
